filebeat/input/filestream/config.go

```diff
@@ -34,7 +34,7 @@ type config struct {
 
 	Paths          []string                `config:"paths"`
 	Close          closerConfig            `config:"close"`
-	FileWatcher    *common.ConfigNamespace `config:"file_watcher"`
+	FileWatcher    *common.ConfigNamespace `config:"prospector"`
```

Comment:
is `watcherDebugKey` in `filebeat/input/filestream/fswatch.go` related

Reply:
No.